packages/backend/src/projectAdapters/projectAdapter.ts

```diff
@@ -2,10 +2,14 @@ import { DbtProjectConfig, ProjectType } from '../config/parseConfig';
 import { ProjectAdapter } from '../types';
 import { DbtLocalProjectAdapter } from './dbtLocalProjectAdapter';
 import { DbtRemoteProjectAdapter } from './dbtRemoteProjectAdapter';
+import { DbtCloudIdeProjectAdapter } from './dbtCloudIdeProjectAdapter';
 
 export const projectAdapterFromConfig = (
     config: DbtProjectConfig,
+    // eslint-disable-next-line consistent-return
 ): ProjectAdapter => {
+    console.log(config);
+    // eslint-disable-next-line default-case
```

Comment:
fyi: These two rules can be solved by returning an error on switch default case.

Reply:
fixed